test/support/autoinst_profile_examples.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
```

Comment:
why file is called profile_examples, when it is section-examples?

Reply:
Because the module is called AutoinstProfile.

Reply:
so it is common behavior for everything in given namespace, right?

Reply:
BTW, I have renamed to a hopefully a better name :)